src/configurations/destinations/zoho_dev/db-config.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "ZOHO_DEV",
+  "displayName": "ZOHO_DEV",
+  "config": {
+    "supportedAccountDefinitions": {
+      "rudderAccountId": ["DESTINATION_ZOH__DEV_OAUTH"]
```

Comment:
`DESTINATION_ZOHO_DEV_OAUTH` is the right account name - also check why did the test case didn't fail ?

Reply:
The command to find files with differences was wrong. Updated to use the common one.